controllers/projects.js

```diff
@@ -0,0 +1,24 @@
+const projectService = require('../services/projects');
+
+module.exports = {
+  async createProject(req, res, next) {
+    try {
+      const { name, description, duration, developers, projectType, hourlyRate, projectValue, endDate, salesChannel, isFinished, } = req.body;
+      const project = await projectService.createProject(name, description, duration, developers, projectType, 
```

Comment:
instead of sending req.body. You send 10 parameters. You can send req.body and in service use it like data.name, data.description ... Also you need validation for this. @ElvaRivle create good example of validation in middleware so you can use his code as example.